.github/workflows/parallel.yml

```diff
@@ -92,6 +92,11 @@ jobs:
       if: ${{ failure() && inputs.summary != '' }}
       run: ${{ inputs.summary }}
       continue-on-error: true
+    - name:  Run Integration Tests and Check test status
```

Comment:
This test should only be executed when we enable the profile of K8S.

Reply:
You can see https://github.com/apache/incubator-uniffle/blob/791e3e09e386391a3e000a9a9b005bc3f4fe8ab3/.github/workflows/build.yml#L86

Reply:
> You can see
> 
> https://github.com/apache/incubator-uniffle/blob/791e3e09e386391a3e000a9a9b005bc3f4fe8ab3/.github/workflows/build.yml#L86



> This test should only be executed when we enable the profile of K8S.

Moving this to the single.yaml should solve this right @jerqi ?

Reply:
> > You can see
> > https://github.com/apache/incubator-uniffle/blob/791e3e09e386391a3e000a9a9b005bc3f4fe8ab3/.github/workflows/build.yml#L86
> 
> > This test should only be executed when we enable the profile of K8S.
> 
> Moving this to the single.yaml should solve this right @jerqi ?

No. We should add this test to build.yml like
```
kubernetes:
.....
kubernetes-integration:
1. we should compile kubernetes operator first: mvn package -Pkubernetes -DskipTests
2. we run the shell to run the local shell tests
.....
```